static/js/vue-cdr-access/src/components/browseDisplay.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+    <div class="browse-records-display">
+        <div v-if="numberOfRecords > 0" class="columns">
+            <div class="column is-10">
+                <browse-search :record-id="container_metadata.id"
+                               @browse-query-results="browseSearching">
+                </browse-search>
+            </div>
+            <div class="column is-2">
+                <browse-sort :records="displayList"
+                             @sort-ordering="sortOrdering">
+                </browse-sort>
+            </div>
+        </div>
+        <div class="columns">
+            <div class="column is-10 spacing">
+                <p :class="{ no_results: numberOfRecords === 0}">
+                    There are <strong>{{ numberOfRecords }}</strong> {{ childTypeText }} in this level.
+                </p>
+                <p v-if="numberOfRecords > 0">
+                    Displaying <strong>{{ pagination_settings.start + 1}}</strong> to
+                    <strong>{{ displayNumber }}</strong> of <strong>{{ numberOfRecords }}</strong>
```

Comment:
I don't think the mockups have this component in it, they just state "There are X items in here", so i think you can drop this line and just leave things up to the pagination to tell people where they are in the result.